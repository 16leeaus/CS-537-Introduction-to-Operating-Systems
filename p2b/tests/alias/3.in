alias ls /bin/ls tests
alias
ls
alias ls /bin/cat tests/3.desc
alias
ls
exit
