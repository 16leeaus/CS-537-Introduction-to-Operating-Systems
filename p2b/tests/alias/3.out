alias ls /bin/ls tests
alias
ls /bin/ls tests
ls
10.desc
10.err
10.in
10.out
10.rc
10.run
11.desc
11.err
11.in
11.out
11.rc
11.run
12.desc
12.err
12.in
12.out
12.rc
12.run
13.desc
13.err
13.in
13.out
13.rc
13.run
14.desc
14.err
14.in
14.out
14.rc
14.run
1.desc
1.err
1.in
1.out
1.rc
1.run
2.desc
2.err
2.in
2.out
2.rc
2.run
3.desc
3.err
3.in
3.out
3.rc
3.run
4.desc
4.err
4.in
4.out
4.rc
4.run
5.desc
5.err
5.in
5.out
5.rc
5.run
6.desc
6.err
6.in
6.out
6.rc
6.run
7.desc
7.err
7.in
7.out
7.rc
7.run
8.desc
8.err
8.in
8.out
8.rc
8.run
9.desc
9.err
9.in
9.out
9.rc
9.run
alias ls /bin/cat tests/3.desc
alias
ls /bin/cat tests/3.desc
ls
Create then replace alias
exit
